lib/dtutils/string.lua

```diff
@@ -192,33 +192,28 @@ dtutils_string.libdoc.functions["sanitize"] = {
   Copyright = [[]],
 }
 
-function dtutils_string.sanitize(str)
-  local result = str
-  local os_quote = dt.configuration.running_os == "windows" and '"' or "'"
-  local escaped_os_quote = "\\" .. os_quote
+function dtutils_string.sanitize_posix(str)
```

Comment:
Why don't we declare this as 
`local function _sanitize_posix(str)` which will make this a private function for the library.